internal/policy/src/main/resources/policy/checkstyle.xml

```diff
@@ -123,27 +129,15 @@
             <property name="allowEmptyTypes" value="true"/>
         </module>
 
-        <module name="RegexpSinglelineJava">
```

Comment:
why would we remove the tab check?

Reply:
Checkstyle offers a built-in check to achieve this: https://checkstyle.sourceforge.io/config_whitespace.html#FileTabCharacter (added new towards top of `checkstyle.xml`).

It's stricter than the `RegexpSinglelineJava` I removed, as it disallows tab characters alltogether. Note that this won't affect the tab constant `\t`.